infra/gcp/terraform/modules/monitoring/uptime-alert/versions.tf

```
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 terraform {
-  required_version = "~> 1.2.0"
+  required_version = ">= 1.2.0"
```

I think it's fine we stick to one major version:
```suggestion
  required_version = "~> 1.3.0"
```

we want to be consistent across the different projects.

---

I didn't want to force the rest of the repo to upgrade yet.